ext/spl/php_spl.stub.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/* ext/spl/php_spl.c */
+
+function class_implements($what, bool $autoload = true) : array|false { }
+
+function class_parents($instance, bool $autoload = true) : array|false {}
+
+function class_uses($what, bool $autoload = true) : array|false {}
+
+function spl_autoload(string $class_name, string $file_extensions = '.inc,.php') : void {}
+
+// This silently ignores non-string class names
+function spl_autoload_call($class_name) : void {}
+
+function spl_autoload_extensions(string $file_extensions = '') : string {}
+
+function spl_autoload_functions() : array|false {}
+
+function spl_autoload_register($autoload_function = null, bool $throw = true, bool $prepend = false) : bool {}
```

Comment:
Not quite right, because the function uses PARSE_PARAMS_QUIET. Not sure why it does that though...

Reply:
Removed in https://github.com/php/php-src/commit/05e6a11b5bf2cc62e0fd10fcc3a6035c3add6c45.